parts/k8s/windowskubeletfunc.ps1

```diff
@@ -183,6 +183,40 @@ Get-KubeBinaries
     Expand-Archive -path $zipfile -DestinationPath C:\
 }
 
+function
+Get-KubeBinaries
+{
+    Param(
+        [Parameter(Mandatory=$true)][string]
+        $KubeBinariesURL
+    )
+
+    if ($computerInfo.WindowsVersion -eq "1709") 
+    {        
+        Write-Log "Server version 1709 does not support using kubernetes binaries in tar file."    
+        return
+    }
+    
+    $binaryPackage = "c:\k.tar.gz"

```

Comment:
I think we should avoid putting more stuff in non-ACL'd and root folders. Can you move this to a temp dir? https://stackoverflow.com/a/34560029

Reply:
the other case of this in `Get-KubernetesPackage` can be removed later. It depends on being extracted from `c:\k.zip`

Reply:
Made this update.